src/FSLibrary/StellarCoreCfg.fs

```diff
@@ -321,7 +321,7 @@ type StellarCoreCfg =
         let invList =
             match self.invariantChecks with
             | AllInvariants -> [ ".*" ]
-            | AllInvariantsExceptBucketConsistencyChecks -> [ "(?!BucketListIsConsistentWithDatabase).*" ]
```

Comment:
The `AllInvariants` need to either enable the tests, or disable the invariant... Maybe we can go with disabling the invariant for now.